modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,286 @@
+| *Summary*: The cluster info cache refresh has been optimized by querying the buckets.uri endpoint only on a change in version hash.
+
+| https://issues.couchbase.com/browse/MB-43766[MB-43766^]
+| *Summary*: The index build tokens were cleaned up only during rebalance. This has been updated so the index build tokens are cleaned up periodically on index deletion and by the lifecycle manager's janitor as well.
+
+| https://issues.couchbase.com/browse/MB-43764[MB-43764^]
+| *Summary*: Fixed an issue where the partition index order was not honored when index projection included all keys and the document key.
+
+| https://issues.couchbase.com/browse/MB-43280[MB-43280^]
+| *Summary*: Fixed an issue where `listReplicaCount` took more than 10s and timed out.
+
+| https://issues.couchbase.com/browse/MB-43072[MB-43072^]
+| Fixed a delay in the connectBucket function that caused the indexer to projector connection time out. 
+|===
+
+==== Query Service
+
+[#table_fixedissues_v662-query,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-45273[MB-45273^]
+| *Summary*: Fixed an issue where the primary index scan was incorrectly used with query containing an OR condition.
+
+| https://issues.couchbase.com/browse/MB-44979[MB-44979^]
+| *Summary*: Connections that timeout on read are now discarded.
+
+| https://issues.couchbase.com/browse/MB-44331[MB-44331^]
+| *Summary*: Fixed an issue where ANSI JOIN with intersect scan did not return any results. 
+
+| https://issues.couchbase.com/browse/MB-43488[MB-43488^]
+| *Summary*: Fixed an issue where an adaptive index with UNNEST alias returned wrong results.
+
+| https://issues.couchbase.com/browse/MB-43384[MB-43384^]
+| *Summary*: Fixed an issue where setupSSL() failed and impacted query execution.
+
+| https://issues.couchbase.com/browse/MB-43301[MB-43301^]
+| *Summary*: Fixed an issue where bulk DELETE with USE KEYS did not delete all keys.
+|===
+
+==== Search Service
+
+[#table_fixedissues_v662-search,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-44485[MB-44485^]
+| *Summary*: Fixed a rebalance failure caused due to a hash mismatch between plan and directory.
+
+| https://issues.couchbase.com/browse/MB-43423[MB-43423^]
+| *Summary*: Fixed an issue where the rebalance stats monitor failed under heavy load.
+
+| https://issues.couchbase.com/browse/MB-43421[MB-43421^]
+| *Summary*: Fixed an issue where indexes were not deleted upon bucket deletion.
+
+| https://issues.couchbase.com/browse/MB-42989[MB-42989^]
+| *Summary*: Fixed the incorrect initialisation of partition UUID with consistency vector search requests.
+|===
+
+==== Tools, Web Console (UI), and REST API
+
+[#table_fixedissues_v662-tools-ui-rest-api,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-44451[MB-44451^]
+| *Summary*: The Admin console now displays a tooltip to help distinguish very long bucket names.
```

Comment:
Also could we add...

MB-44580
Summary: Fixes an issue to ensure the flag -x uncompress=1 operates as expected when using cbtransfer to transfer data out of a cluster.